roles/nvidia-gpu-operator/defaults/main.yml

```diff
@@ -9,23 +9,23 @@
 gpu_operator_helm_repo: "https://nvidia.github.io/gpu-operator"
 gpu_operator_chart_name: "nvidia/gpu-operator"
 gpu_operator_release_name: "nvidia-gpu-operator"
-gpu_operator_chart_version: "1.6.0"
+gpu_operator_chart_version: "1.6.2"
 k8s_gpu_mig_strategy: "mixed"
 
+# Configuration customization
 # XXX: This currently appears to be hardcoded in the operator
 gpu_operator_namespace: "gpu-operator-resources"
+gpu_operator_grid_config_dir: "{{ deepops_dir }}/gpu_operator"
 
 # Defaults from https://github.com/NVIDIA/gpu-operator/blob/master/deployments/gpu-operator/values.yaml
 gpu_operator_default_runtime: "docker"
 gpu_operator_driver_registry: "nvcr.io/nvidia"
 gpu_operator_driver_version: "460.32.03"
 gpu_operator_plugin_args: "--mig-strategy={{ k8s_gpu_mig_strategy }},--pass-device-specs=true,--fail-on-init-error=true,--device-list-strategy=envvar,--nvidia-driver-root=/run/nvidia/driver"
 
-# Variables used if driver containers are in a private registry (i.e. beta drivers & vGPU)
-## Flag that determines whether to add a k8s secret for private containers
-gpu_operator_create_secret: false
-## Name of k8s secret (typically "" or "registry-secret")
-gpu_operator_registry_secret: ""
+# Variables used for vGPU
+## This is the IP of the license server used for vGPU, must be set to use vGPU
+vgpu_grid_license_server: ""
```

Comment:
Consider adding this to config.example/all.yml or config.example/k8s-cluster.yml